Missionframework/modules/11_permission/fnc/fn_permission_changePermission.sqf

```diff
@@ -0,0 +1,81 @@
+/*
+    KPLIB_fnc_permission_changePermission
+
+    File: fn_permission_changePermission.sqf
+    Author: KP Liberation Dev Team - https://github.com/KillahPotatoes
+    Date: 2018-12-16
+    Last Update: 2018-12-28
+    License: GNU General Public License v3.0 - https://www.gnu.org/licenses/gpl-3.0.html
+
+    Description:
+        Changes the selected permission of the given player.
+
+    Parameter(s):
+        _args - Provided arguments of the EH [ARRAY, defaults to []]]
+
+    Returns:
+        Function reached the end [BOOL]
+*/
+
+params [
+    ["_args", [], [[]]]
+];
+
+_args params ["_control"];
+
+// Dialog controls
+private _dialog = findDisplay 758011;
+private _ctrlPlayerList = _dialog displayCtrl 68740;
+
+// Colors
+private _red = [0.9,0,0,1];
+private _green = [0,0.8,0,1];
+private _white = [1,1,1,1];
+
+// Read the control
+(_control getVariable ["Data", ["", false]]) params [
+    "_permission",
+    "_state"
+];
+
+// Get the listbox data
+private _index = lbCurSel _ctrlPlayerList;
+private _playerUID = _ctrlPlayerList lbData _index;
+private _playerArray = [];
+private _playerPermissions = [];
+private _permissionArray = [];
+
```

Comment:
Another small thing:
Maybe you could add a `_permission = toLower _permission;` here and remove that `toLower` in the code below. In the end it's still only 1 `toLower` which is executed in the whole code, but personally I think it would be nicer to read. This applies also to other functions where the `toLower _permission` is used more than 1 time.